### PR TITLE
HTS-1137: Added the threshold to the eligibility response from the BE

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/models/eligibility/EligibilityCheckResult.scala [hunk 1]
```diff
@@ -18,35 +18,18 @@ package uk.gov.hmrc.helptosavefrontend.models.eligibility
...
-  case class Ineligible(value: EligibilityCheckResponse) extends EligibilityCheckResult
+  implicit val format: Format[EligibilityCheckResult] = Json.format[EligibilityCheckResult]
 
-  case class AlreadyHasAccount(value: EligibilityCheckResponse) extends EligibilityCheckResult
+}
 
-  object Eligible {
-    implicit val format: Format[Eligible] = Json.format[Eligible]
-  }
+case class EligibilityCheckResponseAndThreshold(eligibilityCheckResult: EligibilityCheckResult, threshold: Option[Double])
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/helptosavefrontend/models/eligibility/EligibilityCheckResult.scala]: in the backend logic, this class is `EligibilityCheckResponse` , better to keep the same naming here to keep consistency

File: conf/messages [hunk 2]
```diff
@@ -229,7 +229,12 @@ hts.core.not-eligible.wtc.p-4-1=Find out what to do if you have checked the elig
 hts.core.not-eligible.wtc.p-4-link-text=think you&rsquo;re eligible
 hts.core.not-eligible.wtc.p-4-2=for a Help to Save account.
 
-hts.core.not-eligible.uc.p-1=This is because your household income - in your last monthly assessment period - was less than £542.88. Your Universal Credit payments are not considered to be income.
+hts.core.not-eligible.uc.p-1=This is because your household income - in your last monthly assessment period
+hts.core.not-eligible.uc.p-1b=This is because your household income in your last monthly assessment period
+hts.core.not-eligible.uc.p-1-2a=- was less than £
+hts.core.not-eligible.uc.p-1-2b= was not enough.
```
Comment (sureshhmrc) [inline conf/messages]: is extra space after `=` necessary ?

Comment (Jackie1988) [reply conf/messages]: no it wasn't necessary, I have now removed it.

File: app/uk/gov/hmrc/helptosavefrontend/models/eligibility/EligibilityCheckResult.scala (hunk 1 above)
Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosavefrontend/models/eligibility/EligibilityCheckResult.scala]: ok I've changed it to that, thanks

**Pull request merged**